sdk/typescript/src/keypairs/ed25519/keypair.ts

```diff
@@ -63,17 +61,26 @@ export class Ed25519Keypair extends Keypair {
 	 * This is NOT the private scalar which is result of hashing and bit clamping of
 	 * the raw secret key.
 	 *
-	 * The sui.keystore key is a list of Base64 encoded `flag || privkey`. To import
-	 * a key from sui.keystore to typescript, decode from base64 and remove the first
-	 * flag byte after checking it is indeed the Ed25519 scheme flag 0x00 (See more
-	 * on flag for signature scheme: https://github.com/MystenLabs/sui/blob/818406c5abdf7de1b80915a0519071eec3a5b1c7/crates/sui-types/src/crypto.rs#L1650):
+	 * The sui.keystore key is a list of Bech32 encoded `flag || privkey`. To import
+	 * a key from sui.keystore to typescript, decode from bech32 and parse to the
+	 * corresponding keypair based on scheme.
 	 * ```
-	 * import { Ed25519Keypair, fromB64 } from '@mysten/sui.js';
-	 * const raw = fromB64(t[1]);
-	 * if (raw[0] !== 0 || raw.length !== PRIVATE_KEY_SIZE + 1) {
-	 *   throw new Error('invalid key');
+	 * 	const {prefix, words} = bech32.decode(keypair.privateKey);
```

Comment:
We should export a helper to do the bech32 parse of the private key, so that users don't need to add the dependency and write this code.

Reply:
added to util: https://github.com/MystenLabs/sui/blob/83d0fb734804336e3ecc255f1a1f6bd89ef56913/sdk/typescript/src/utils/sui-types.ts is this appropriate?